views/partials/_whats-new.njk

```diff
@@ -6,9 +6,8 @@
     <div class="govuk-grid-row">
       <div class="govuk-grid-column-two-thirds-from-desktop">
         <h2 id="whats-new" class="govuk-heading-l">What’s new</h2>
-<p class="govuk-body">18 August 2022: We’ve released GOV.UK Frontend v4.3.1. This fix release includes a change to the naming of our static spacing override classes.</p>
-        <p class="govuk-body"><a href="https://github.com/alphagov/govuk-frontend/releases/tag/v4.3.1" class="govuk-link">Read the release notes to see what changes you should make</a>.</p>
-
+<p class="govuk-body">17 October 2022: We’ve changed our ‘Backlog’ page into the new '<a href="https://design-system.service.gov.uk/community/upcoming-components-patterns/">Upcoming components and patterns</a>' page and chosen 3 priorities that we plan to work on next.
```

Comment:
This should be a relative link rather than a full URL:

```suggestion
<p class="govuk-body">17 October 2022: We’ve changed our ‘Backlog’ page into the new '<a href="/community/upcoming-components-patterns/">Upcoming components and patterns</a>' page and chosen 3 priorities that we plan to work on next.
```